src/main/java/mil/dds/anet/resources/ReportResource.java

```diff
@@ -208,6 +209,7 @@ private Person findPrimaryAttendee(Report r, boolean isInterlocutor) {
    * @return the report as it was stored in the database before this method was called.
    */
   private Report executeReportUpdates(Person editor, Report r) {
+    ResourceUtils.assertAllowedClassification(r.getClassification());
```

Comment:
I would do this assertion _after_ asserting that the user can actually update the report.